db/migrate/20190710075926_add_name_to_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddNameToUsers < ActiveRecord::Migration[4.2]
+  def change
+    add_column :users, :name, :string
+    add_column :users, :date_of_birth, :datetime
```

Comment:
why did you add these other columns?

Reply:
We added this when we were trying to add more requirement fields to the sign up form, because Devise only has "email" and "password" and we also wanted to add "name". we watched some tutorials and ended up adding more fields like date of birth, gender and all that. But at the end we are only using the name, so all the excess is now removed.